docker-compose.yml

```diff
@@ -106,10 +106,15 @@ services:
       - "1025"
       - "8025"
     labels:
-      - "traefik.enable=true"
-      - "traefik.docker.network=frontend"
       - "traefik.http.routers.${COMPOSE_PROJECT_NAME}-http.rule=Host(`${COMPOSE_DOMAIN}`)"
 
+  redis:
```

Comment:
I think redis should be added to docker-compose.server.override.yml as well to work for stg and prod.

Reply:
Added